src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/EnvelopeProcessorValidationTest.java

```diff
@@ -76,6 +77,31 @@ public void should_throw_exception_when_zip_file_has_mismatching_pdf() throws Ex
             .hasMessageContaining("Missing PDFs: 1111001.pdf");
     }
 
+    @Test
+    public void should_throw_exception_when_ocr_data_is_missing_for_sscs_new_application() throws Exception {
```

Comment:
It would also be worth adding a test for BlobProcessorTask, checking if this method is used.

Reply:
We don't have a unit test for BlobProcessorTask. I think it's better to do it as part of another ticket